src/pudl/metadata/fields.py

```diff
@@ -882,6 +978,22 @@
             "different."
         ),
     },
+    "emission_control_id_eia": {
+        "type": "string",
+        "description": (
+            "The emission control ID used to collect so2, nox, particulate, "
```

Comment:
Nit: I think this didn't even come from your branch, but it should be SO2 and NOx